lang/en.yml

```diff
@@ -164,8 +164,8 @@ en:
     JOB_STALLED: 'Job stalled after {attempts} attempts - please check'
     MEMORY_RELEASE: 'Job releasing memory and waiting ({used} used)'
     STALLED_JOB: 'Stalled job'
-    STALLED_JOB_MSG: 'A job named {name} appears to have stalled. It has been paused, please login to check it'
-    STALLED_JOB_RESTART_MSG: 'A job named {name} appears to have stalled. It will be stopped and restarted, please login to make sure it has continued'
+    STALLED_JOB_MSG: 'A job named {name} (#{id}) appears to have stalled. It has been paused, please login to check it'
+    STALLED_JOB_RESTART_MSG: 'A job named {name} (#{id}) appears to have stalled. It will be stopped and restarted, please login to make sure it has continued'
```

Comment:
Having a new item like that might cause problems in older versions. Need to check.